02_activities/assignments/assignment.sh

```diff
@@ -9,16 +9,39 @@ set -x
 
 # 1. Create 5 directories named dir1, dir2, dir3, dir4, and dir5
 
-# 2. List the contents of the parent directory to verify the presence of the 5 directories
+mkdir dir1 dir2 dir3 dir4 dir5
 
-# 3. Create 5 text files in dir2 named file1, file2, file3, file4, and file5
+# 2. List the contents of the current working directory to verify the presence of the 5 directories
 
-# 4. Append the words "Hello world" to dir2/file3
+ls
 
-# 5. Verify that file3 contains the words "hello world" by printing the contents of the file in the terminal
+# 3. Create 5 text files in dir2 named file1.txt, file2.txt, file3.txt, file4.txt, and file5.txt
 
-# 6. Delete file4
+cd dir2
+touch file1.txt
+touch file2.txt
+touch file3.txt file4.txt file5.txt
+
+# 4. Append the words "Hello world" to dir2/file3.txt
+
+echo "Hello World" >> file3.txt
+
+# 5. Verify that file3.txt contains the words "hello world" by printing the contents of the file in the terminal
+
+cat file3.txt
+
+# 6. Delete file4.txt
+
+rm file4.txt
 
 # 7. Delete directories dir4 and dir5 including all their contents (if any)
 
-# 8. List the contents of the parent directory to verify the deletion of dir4 and dir5
+cd ../
```

Comment:
For better practice, remove cd dir2 in question 3 and define the relative path when operating on file or directories (eg. cat dir2/file3.txt)